spec/rails_helper.rb

```diff
@@ -61,6 +61,7 @@
   # database_cleaner configuration
   # Clear the entire DB before tests begin
   config.before(:suite) do
+    DatabaseCleaner.allow_remote_database_url = true
```

Comment:
Can you add an explanation as to why we need to allow remote db url for our tests?

Reply:
in http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fgibct-data-service/detail/rails_5.0.7.2/18/pipeline as well as few other builds received the error below

```
An error occurred in a `before(:suite)` hook.
Failure/Error: DatabaseCleaner.clean_with(:truncation)

DatabaseCleaner::Safeguard::Error::RemoteDatabaseUrl:
  ENV['DATABASE_URL'] is set to a remote URL. Please refer to https://github.com/DatabaseCleaner/database_cleaner#safeguards
# /usr/local/bundle/gems/database_cleaner-1.7.0/lib/database_cleaner/safeguard.rb:21:in `run'
# /usr/local/bundle/gems/database_cleaner-1.7.0/lib/database_cleaner/safeguard.rb:69:in `block in run'
# /usr/local/bundle/gems/database_cleaner-1.7.0/lib/database_cleaner/safeguard.rb:69:in `each'
# /usr/local/bundle/gems/database_cleaner-1.7.0/lib/database_cleaner/safeguard.rb:69:in `run'
# /usr/local/bundle/gems/database_cleaner-1.7.0/lib/database_cleaner/base.rb:19:in `initialize'
# /usr/local/bundle/gems/database_cleaner-1.7.0/lib/database_cleaner/configuration.rb:46:in `new'
# /usr/local/bundle/gems/database_cleaner-1.7.0/lib/database_cleaner/configuration.rb:46:in `connections'
# /usr/local/bundle/gems/database_cleaner-1.7.0/lib/database_cleaner/configuration.rb:91:in `clean_with'
# ./spec/rails_helper.rb:64:in `block (2 levels) in <top (required)>'
```

Reply:
Could we do something like:

```suggestion
    DatabaseCleaner.allow_remote_database_url = ENV['CI']
```
This should safeguard against accidentally connecting to remote db in development, to avoid a footgun